Quaver.Shared/Screens/Gameplay/Rulesets/Keys/GameplayRulesetKeys.cs

```diff
@@ -78,109 +78,93 @@ private void InitializeTimingLines()
                         {
-                            switch (ConfigManager.ScrollDirection4K.Value)
-                            {
-                                case ScrollDirection.Down:
-                                    {
-                                        TimingLineManager.Add(new TimingLineManager
-                                        (
-                                            this,
-                                            ScrollDirection.Down,
-                                            playfield.HitPositionOffsets[0],
-                                            playfield.Width,
-                                            0
-                                        ));
-                                        break;
-                                    }
-                                case ScrollDirection.Up:
-                                    {
-                                        TimingLineManager.Add(new TimingLineManager
-                                        (
-                                            this,
-                                            ScrollDirection.Up,
-                                            playfield.HitPositionOffsets[0],
-                                            playfield.Width,
-                                            0
-                                        ));
-                                        break;
-                                    }
-                                case ScrollDirection.Split:
-                                    {
-                                        var halfway = playfield.Stage.Receptors[1].X + playfield.Stage.Receptors[1].Width;
-                                        TimingLineManager.Add(new TimingLineManager
-                                        (
-                                            this,
-                                            ScrollDirection.Down,
-                                            playfield.HitPositionOffsets[0],
-                                            halfway,
-                                            0
-                                        ));
-                                        TimingLineManager.Add(new TimingLineManager
-                                        (
-                                            this,
-                                            ScrollDirection.Up,
-                                            playfield.ColumnLightingPositionY[2],
-                                            playfield.Width - halfway,
-                                            playfield.Width - halfway
-                                        ));
-                                        break;
-                                    }
-                            }
-                            break;
+                            case ScrollDirection.Down:
+                                TimingLineManager.Add(new TimingLineManager
```

Comment:
`TimingLineManager.Add(new TimingLineManager(this, ScrollDirection.Down, playfield.HitPositionOffsets[0], playfield.Width, 0));`

Use one line. The extra line breaks aren't necessary and make it difficult to read. This applies for the other ones as well.

Reply:
Instead of doing it like this, why not use object initialization to add the timing lines instead of calling Add each time. It should do the same thing in a much cleaner way.

Reply:
> Instead of doing it like this, why not use object initialization to add the timing lines instead of calling Add each time. It should do the same thing in a much cleaner way.

Can you elaborate what you mean by this? I don't quite understand.

Reply:
I think I understood what you meant and implemented the method differently. Coffee wise, everything looks much cleaner.